lib/2fa.coffee

```diff
@@ -78,7 +84,13 @@ get2fa = (deps, opts) ->
 	# });
 	###
 	exports.isPassed = (callback) ->
-		token.getProperty('twoFactorRequired').then (twoFactorRequired) ->
+		auth.getType().then (tokenType) ->
+			if tokenType != 0
+				# API key
+				return true
+			else
+				# JWT
+				return auth.needs2FA().then (twoFactorRequired) ->
 			return not twoFactorRequired
```

Comment:
The indentation here is confusing. This line should be more indented then the one above, right? What's going on here?

In addition, I think you can probably skip this one too, because `needs2FA` already handles this, probably (we should check though, for both these cases)